plugins/filter_aws/aws.h

```diff
@@ -106,11 +107,17 @@ struct flb_filter_aws {
     size_t account_id_len;
     int account_id_include;
 
-
     flb_sds_t hostname;
     size_t hostname_len;
     int hostname_include;
 
+    flb_sds_t *tag_keys;
+    size_t *tag_keys_len;
+    flb_sds_t *tag_values;
+    size_t *tag_values_len;
```

Comment:
I think this can also just be a size_t, right?

Reply:
Its an array of size_t since there are many tags. This confused me at first too.

Reply:
I added a comment to improve readability and avoid potential misunderstanding by others, https://github.com/fluent/fluent-bit/pull/6065/commits/905cfbad0b764068eefeceb99e6bf79d34247004